biome.json

```diff
@@ -7,7 +7,7 @@
     },
     "files": {
         "ignoreUnknown": false,
-        "ignore": ["**/dist/*", "**/coverage/*"]
+        "ignore": ["**/dist/*", "**/coverage/*", "**/.astro/*"]
```

Comment:
There are some internal `.astro` hidden folders that we want Biome to ignore for formatting and linting